deploy/olm-catalog/eclipse-che-preview-kubernetes/manifests/che-operator.clusterserviceversion.yaml

```diff
@@ -13,12 +13,12 @@ metadata:
           "spec": {
             "k8s": {
                 "ingressDomain": "",
-                "tlsSecretName": ""
+                "tlsSecretName": "che-tls"
```

Comment:
You provided this changes only for kuberentes csv file. We have openshift csv also.

Reply:
This property is under `k8s` group.

Reply:
It is not used in OpenShift

Reply:
Ok, sorry, but what about lines with set up devfileRegistryImage and devfileRegistryImage to empty value? https://github.com/eclipse/che-operator/pull/508/files#diff-d45591b58d736ef67fb4c34a130def83b8461a9f44998f0a3bae6d9aee239e66R20 for example